src/PeopleSDKAdapter.js

```diff
@@ -0,0 +1,107 @@
+import {Observable} from 'rxjs';
+import {PeopleAdapter, PersonStatus} from '@webex/component-adapter-interfaces';
+
+export const USER_PRESENCE_UPDATE_EVENT = 'event:apheleia.subscription_update';
+
+/**
+ * The `PeopleSDKAdapter` is an implementation of the `PeopleAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to fetch data about a person.
+ *
+ * @export
+ * @class PeopleSDKAdapter
+ * @extends {PeopleAdapter}
+ */
+export default class PeopleSDKAdapter extends PeopleAdapter {
+  /**
+   * Returns a proper Enum key for a given PersonStatus value
+   *
+   * @param {string} status
+   * @returns {PersonStatus.<String>}
+   * @memberof PeopleSDKAdapter
+   */
+  getStatus(status) {
+    return Object.keys(PersonStatus).find((key) => PersonStatus[key] === status);
+  }
+
+  /**
+   * Encodes the personID into base64 format and return the UUID
+   *
+   * @param {string} ID
+   * @returns {string}
+   * @memberof PeopleSDKAdapter
+   */
+  getPersonUUID(ID) {
+    return Buffer.from(ID, 'base64')
+      .toString()
+      .split('/')[4];
+  }
+
+  async getPersonPresence(ID) {
+    try {
+      const {statusList} = await this.datasource.internal.presence.list([this.getPersonUUID(ID)]);
+
+      return this.getStatus(statusList[0].status);
+    } catch (error) {
+      throw new Error(error);
+    }
+  }
+
+  /**
+   * Fetches the person data from the sdk and returns in the shape required by adapter.
+   *
+   * @param {string} ID
+   * @returns {Room}
+   * @memberof RoomsSDKAdapter
+   */
+  async fetchPerson(ID) {
+    try {
+      const {id, emails, displayName, firstName, lastName, avatar, orgId} = await this.datasource.people.get(ID);
+
+      return {
+        ID: id,
+        emails,
+        displayName,
+        firstName,
+        lastName,
+        avatar,
+        orgID: orgId,
+        status: await this.getPersonPresence(ID),
+      };
+    } catch (error) {
+      throw new Error(error);
+    }
+  }
+
+  /**
+   * Returns an observable that emits person data of the given ID.
+   *
+   * @param {string} ID ID of person to get.
+   * @returns {Observable.<Person>}
+   * @memberof PeopleSDKAdapter
+   */
+  getPerson(ID) {
+    // eslint-disable-next-line consistent-return
+    return Observable.create((observer) => {
+      if (!ID) {
+        observer.error(new Error(`Could not find person with ID "${ID}"`));
+      } else {
+        this.fetchPerson(ID)
+          .then((person) => {
+            observer.next(person);
+
+            this.datasource.internal.mercury.on(USER_PRESENCE_UPDATE_EVENT, (event) => {
```

Comment:
You haven't done a `presence.subscribe` for this person, so you will not get any of these events sent to your device.

Reply:
I'm not following this.